src/modules/completeStatus.js

```diff
@@ -16,11 +14,10 @@ export const markAsCompleted = (todo, listItem) => {
     setData();
   }
 };
-
+// finally got it to work
 export const completed = (todo) => {
```

Comment:
- [ ] I would strongly suggest you to delete all unused codes in your project such as `console.log`. I can see them in your `newTask.js` file.